e2e/framework/manager/rook_test_infra.go

```diff
@@ -270,7 +272,9 @@ func (r *rookTestInfraManager) getContainerIdByName(containerName string) (conta
 
 //method for create rook-operator via kubectl
 func createK8sRookOperator(k8sHelper *utils.K8sHelper, tag string) error {
-	raw, err := ioutil.ReadFile(podSpecPath + "/" + rookOperatorFileName)
+	path := filepath.Join(os.Getenv("GOPATH"), podSpecPath)
```

Comment:
if we used relative paths this would be simpler. Otherwise, I'd suggest a helper to get the pod spec path instead of copying this line of code in several places